sys/tools/syscalls/main.lua

```diff
@@ -0,0 +1,63 @@
+#!/usr/libexec/flua
+--
+-- SPDX-License-Identifier: BSD-2-Clause
+--
+-- Copyright (c) 2024 Tyler Baxter <agge@FreeBSD.org>
+-- Copyright (c) 2023 Warner Losh <imp@bsdimp.com>
+-- Copyright (c) 2019 Kyle Evans <kevans@FreeBSD.org>
+--
+-- Thanks to Kyle Evans for his makesyscall.lua in FreeBSD which served as
+-- inspiration for this, and as a source of code at times.
+--
+
+-- We generally assume that this script will be run by flua, however we've
+-- carefully crafted modules for it that mimic interfaces provided by modules
+-- available in ports.  Currently, this script is compatible with lua from ports
+-- along with the compatible luafilesystem and lua-posix modules.
+
```

Comment:
I'd remove copies of this syscalls/scripts/*.lua. I might also move this copy to the README.md.